test/s3tables/admin/admin_iceberg_pages_test.go

```
@@ -0,0 +1,555 @@
+			"-webdav.port=0",
+			"-master.volumeSizeLimitMB=32",
+			"-ip=127.0.0.1",
+			"-master.peers=none",
+			"-s3.iam.readOnly=false",
+		}
+		os.Args = append([]string{"weed"}, args...)
+		glog.MaxSize = 1024 * 1024
+		for _, cmd := range command.Commands {
+			if cmd.Name() == "mini" && cmd.Run != nil {
+				cmd.Flag.Parse(os.Args[1:])
+				command.MiniClusterCtx = ctx
```

golangci-lint (2.12.2)

[error] nested context in loop

(fatcontext)